data/releases/9.1.md

```diff
@@ -30,7 +30,8 @@ The base proof assistant can be installed as an opam switch with the following c
 ```bash
 opam update
 opam switch create 4.14.1
-opam install rocq-prover.9.1.0
+opam pin add rocq-runtime 9.1.0
+opam install rocq-prover
```

Comment:
```suggestion
opam install rocq-prover rocq-runtime=9.1.0
```
`opam pin` should really be reserved to expert users (and they do not need to be told about it). Most users should just use `opam install`.